Cards/Cardstyle.css

```diff
@@ -0,0 +1,187 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700;900&display=swap');
+
+:root {
+    --primary-color : #c2004b;
+    --secondary-color : #eb679a;
+    --light-color : #ffeff5;
+    --dark-color : #72022d;
+}
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Roboto;
+}
+::-webkit-scrollbar {
+    width: 0px;
+}
+body {
+    background-color: #F7F7F7;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+/* Common css for card */
+.card {
+    border-radius: 4px;
+    width: 350px;
+    margin : 35px 0px;
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    overflow: hidden;
+}
+.card .card-container .card-text {
+    padding: 1rem;
+}
+.card .card-container .card-text .card-heading {
+    font-size : 1.25rem;
+    line-height: 2rem;
+}
+.card .card-container .card-text .card-subheading {
+    font-size: 0.875rem;
+    color: #0000008A;
```

Comment:
preferred to use CSS variables for colors, define them in another stylesheet and use the variable name here using `var()`

Reply:
yes, will do that